config/defaults/config.LINUX_INTEL.mk

```diff
@@ -27,7 +27,8 @@ CGNS_INCLUDE_FLAGS=-I$(CGNS_HOME)/include
 CGNS_LINKER_FLAGS=-L$(CGNS_HOME)/lib -lcgns
 
 # ------- Define Compiler Flags ----------------------------------------
-FF90_FLAGS   = -DHAS_ISNAN -fPIC -r8 -O2 -g
+FF77_FLAGS   = -DHAS_ISNAN -fPIC -r8 -O2 -g
+FF90_FLAGS = $(FF77_FLAGS) -std=f2008
```

Comment:
Doesn't Intel have a different standard flag?

Reply:
Yes, they do. Should be updated now.